### DISCUSSION TITLE
slow download

### DISCUSSION THREAD

**Discussion by ghost** [UltimaHoarder/UltimaScraper#1133] [General]
it just took me 50 minutes to download 1.5gb of pictures, it seems take some time initializing the next download. is there a setting that i can change to make this go faster?

**Comment by Gowersblade** [comment]
Similar problem here - well the file starts off as my expected broadband speed then just goes so slow it would maybe take a day to download 1 file. I've tried using VPN and my normal ISP too - example below. Great work mind. :)

0it [00:00, ?it/s]
  0%|[32m          [0m| 4.10k/39.9M [00:00<2:10:57, 5.08kB/s]
  0%|[32m          [0m| 69.6k/39.9M [00:00<06:29, 102kB/s]   
  0%|[32m          [0m| 221k/576M [00:01<27:20, 351kB/s]  
  0%|[32m          [0m| 238k/576M [00:01<09:58, 962kB/s]
  0%|[32m          [0m| 389k/576M [00:01<09:58, 962kB/s]
  0%|[32m          [0m| 442k/576M [00:01<09:57, 962kB/s]
  0%|[32m          [0m| 549k/576M [00:01<09:57, 962kB/s]
  0%|[32m          [0m| 954k/576M [00:01<06:24, 1.49MB/s]
  0%|[32m          [0m| 979k/576M [00:01<04:05, 2.34MB/s]
  0%|[32m          [0m| 1.12M/576M [00:01<04:05, 2.34MB/s]
  0%|[32m          [0m| 1.16M/576M [00:01<04:05, 2.34MB/s]
  0%|[32m          [0m| 1.23M/576M [00:01<04:05, 2.34MB/s]
  0%|[32m          [0m| 1.33M/576M [00:01<04:05, 2.34MB/s]
  0%|[32m          [0m| 1.39M/576M [00:01<04:05, 2.34MB/s]
  0%|[32m          [0m| 1.72M/927M [00:01<06:57, 2.22MB/s]
  0%|[32m          [0m| 1.74M/927M [00:01<08:12, 1.88MB/s]
  0%|[32m          [0m| 1.91M/927M [00:01<08:12, 1.88MB/s]
  0%|[32m          [0m| 2.02M/927M [00:01<08:12, 1.88MB/s]
  0%|[32m          [0m| 2.07M/927M [00:01<08:12, 1.88MB/s]
  0%|[32m          [0m| 2.10M/927M [00:01<08:12, 1.88MB/s]
  0%|[32m          [0m| 2.11M/927M [00:01<08:12, 1.88MB/s]
  0%|[32m          [0m| 2.33M/927M [00:01<08:26, 1.82MB/s]
  0%|[32m          [0m| 2.34M/927M [00:01<08:51, 1.74MB/s]
  0%|[32m          [0m| 3.31M/927M [00:01<04:32, 3.39MB/s]
  0%|[32m          [0m| 3.34M/927M [00:01<02:20, 6.59MB/s]
  0%|[32m          [0m| 3.36M/927M [00:01<02:20, 6.59MB/s]
  0%|[32m          [0m| 3.60M/927M [00:01<02:20, 6.59MB/s]
  0%|[32m          [0m| 3.66M/927M [00:01<02:20, 6.59MB/s]
  0%|[32m          [0m| 3.72M/927M [00:01<02:20, 6.59MB/s]
  0%|[32m          [0m| 3.93M/927M [00:01<02:20, 6.59MB/s]
  0%|[32m          [0m| 4.11M/927M [00:01<02:19, 6.59MB/s]
  1%|[32m          [0m| 4.97M/927M [00:01<03:16, 4.68MB/s]
  1%|[32m          [0m| 4.98M/927M [00:01<05:04, 3.03MB/s]
  1%|[32m          [0m| 4.99M/927M [00:01<05:04, 3.03MB/s]
  1%|[32m          [0m| 5.00M/927M [00:01<05:04, 3.03MB/s]
  1%|[32m          [0m| 5.38M/927M [00:02<05:37, 2.73MB/s]
  1%|[32m          [0m| 5.40M/927M [00:02<07:38, 2.01MB/s]
  1%|[32m          [0m| 5.43M/927M [00:02<07:38, 2.01MB/s]
  1%|[32m          [0m| 5.53M/927M [00:02<07:38, 2.01MB/s]
  1%|[32m          [0m| 5.77M/927M [00:02<07:38, 2.01MB/s]
  1%|[32m          [0m| 5.82M/927M [00:02<07:38, 2.01MB/s]
  1%|[32m          [0m| 6.04M/927M [00:02<10:35, 1.45MB/s]
  1%|[32m          [0m| 6.32M/927M [00:02<08:07, 1.89MB/s]
  1%|[32m          [0m| 6.44M/927M [00:02<08:07, 1.89MB/s]
  1%|[32m          [0m| 6.94M/927M [00:02<04:58, 3.08MB/s]
  1%|[32m          [0m| 7.05M/927M [00:02<04:58, 3.08MB/s]
  1%|[32m          [0m| 7.21M/927M [00:02<07:34, 2.02MB/s]
  1%|[32m          [0m| 7.35M/927M [00:02<07:34, 2.02MB/s]
  1%|[32m          [0m| 7.53M/927M [00:02<07:34, 2.02MB/s]
  1%|[32m          [0m| 7.74M/927M [00:02<12:20, 1.24MB/s]
  1%|[32m          [0m| 8.04M/927M [00:02<12:39, 1.21MB/s]
  1%|[32m          [0m| 8.17M/927M [00:02<12:39, 1.21MB/s]
  1%|[32m          [0m| 8.28M/927M [00:02<12:39, 1.21MB/s]
  1%|[32m          [0m| 8.51M/927M [00:02<16:53, 906kB/s] 
  1%|[32m          [0m| 8.69M/927M [00:02<16:57, 902kB/s]
  1%|[32m          [0m| 8.83M/927M [00:02<16:57, 902kB/s]
  1%|[32m          [0m| 8.97M/927M [00:02<20:12, 757kB/s]
  1%|[32m          [0m| 9.11M/927M [00:02<23:37, 647kB/s]
  1%|[32m          [0m| 9.22M/927M [00:02<23:37, 647kB/s]
  1%|[32m          [0m| 9.34M/927M [00:03<26:09, 584kB/s]
  1%|[32m          [0m| 9.45M/927M [00:03<29:09, 524kB/s]
  1%|[32m          [0m| 9.54M/927M [00:03<33:22, 458kB/s]
  1%|[32m          [0m| 9.64M/927M [00:03<37:32, 407kB/s]
  1%|[32m          [0m| 9.73M/927M [00:03<41:32, 368kB/s]
  1%|[32m          [0m| 9.81M/927M [00:03<48:10, 317kB/s]
  1%|[32m          [0m| 9.88M/927M [00:03<53:07, 288kB/s]
  1%|[32m          [0m| 9.94M/927M [00:03<57:39, 265kB/s]
  1%|[32m          [0m| 10.0M/927M [00:03<1:03:43, 240kB/s]
  1%|[32m          [0m| 10.1M/927M [00:03<1:11:22, 214kB/s]
  1%|[32m          [0m| 10.1M/927M [00:03<1:21:02, 189kB/s]
  1%|[32m          [0m| 10.2M/927M [00:03<1:31:22, 167kB/s]
  1%|[32m          [0m| 10.2M/927M [00:03<1:42:12, 149kB/s]
  1%|[32m          [0m| 10.2M/927M [00:03<1:53:39, 134kB/s]
  1%|[32m          [0m| 10.3M/927M [00:03<2:07:21, 120kB/s]
  1%|[32m          [0m| 10.3M/927M [00:03<2:26:39, 104kB/s]
  1%|[32m          [0m| 10.3M/927M [00:03<2:44:30, 92.8kB/s]
  1%|[32m          [0m| 10.3M/927M [00:03<3:08:21, 81.1kB/s]
  1%|[32m          [0m| 10.4M/927M [00:03<3:38:46, 69.8kB/s]
  1%|[32m          [0m| 10.4M/927M [00:03<4:07:36, 61.7kB/s]
  1%|[32m          [0m| 10.4M/927M [00:04<4:46:59, 53.2kB/s]
  1%|[32m          [0m| 10.4M/927M [00:04<5:25:03, 47.0kB/s]
  1%|[32m          [0m| 10.4M/927M [00:04<6:00:27, 42.4kB/s]
  1%|[32m          [0m| 10.4M/927M [00:04<6:52:14, 37.0kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<7:44:41, 32.9kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<8:27:21, 30.1kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<9:55:16, 25.7kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<11:15:01, 22.6kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<12:30:56, 20.3kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<13:36:46, 18.7kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<17:42:40, 14.4kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<20:52:21, 12.2kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<20:52:21, 12.2kB/s]
  1%|[32m          [0m| 10.5M/927M [00:04<23:46:54, 10.7kB/s]
  1%|[32m          [0m| 10.5M/927M [00:05<26:27:21, 9.62kB/s]
  1%|[32m          [0m| 10.5M/927M [00:05<28:58:01, 8.79kB/s]
  1%|[32m          [0m| 10.5M/927M [00:05<34:13:02, 7.44kB/s]
  1%|[32m          [0m| 10.5M/927M [00:05<36:46:44, 6.92kB/s]
  1%|[32m          [0m| 10.5M/927M [00:05<36:46:44, 6.92kB/s]
  1%|[32m          [0m| 10.6M/927M [00:05<38:51:15, 6.55kB/s]
  1%|[32m          [0m| 10.6M/927M [00:05<41:04:36, 6.20kB/s]
  1%|[32m          [0m| 10.6M/927M [00:05<43:35:25, 5.84kB/s]
  1%|[32m          [0m| 10.6M/927M [00:05<45:49:49, 5.55kB/s]
  1%|[32m          [0m| 10.6M/927M [00:05<48:54:50, 5.20kB/s]
  1%|[32m          [0m| 10.6M/927M [00:06<49:09:43, 5.18kB/s]
  1%|[32m          [0m| 10.6M/927M [00:06<49:09:43, 5.18kB/s]
  1%|[32m          [0m| 10.6M/927M [00:06<51:00:59, 4.99kB/s]
  1%|[32m          [0m| 10.6M/927M [00:06<51:00:59, 4.99kB/s]
  1%|[32m          [0m| 10.6M/927M [00:06<52:24:31, 4.86kB/s]
  1%|[32m          [0m| 10.6M/927M [00:06<53:42:11, 4.74kB/s]
  1%|[32m          [0m| 10.6M/927M [00:06<53:55:45, 4.72kB/s]
  1%|[32m          [0m| 10.6M/927M [00:07<53:55:45, 4.72kB/s]
  1%|[32m          [0m| 10.6M/927M [00:07<57:22:36, 4.44kB/s]
  1%|[32m          [0m| 10.6M/927M [00:07<69:08:26, 3.68kB/s]
  1%|[32m          [0m| 10.6M/927M [00:07<69:08:26, 3.68kB/s]
  1%|[32m          [0m| 10.6M/927M [00:07<78:37:57, 3.24kB/s]
  1%|[32m          [0m| 10.6M/927M [00:07<81:33:46, 3.12kB/s]
  1%|[32m          [0m| 10.6M/927M [00:08<81:33:46, 3.12kB/s]
  1%|[32m          [0m| 10.6M/927M [00:08<84:44:56, 3.00kB/s]
  1%|[32m          [0m| 10.6M/927M [00:08<86:13:58, 2.95kB/s]
  1%|[32m          [0m| 10.6M/927M [00:08<83:30:56, 3.05kB/s]
  1%|[32m          [0m| 10.6M/927M [00:08<83:30:56, 3.05kB/s]
  1%|[32m          [0m| 10.6M/927M [00:08<73:31:28, 3.46kB/s]
  1%|[32m          [0m| 10.6M/927M [00:08<73:31:28, 3.46kB/s]
  1%|[32m          [0m| 10.7M/927M [00:08<69:44:12, 3.65kB/s]
  1%|[32m          [0m| 10.7M/927M [00:09<64:15:34, 3.96kB/s]
  1%|[32m          [0m| 10.7M/927M [00:09<64:15:34, 3.96kB/s]
  1%|[32m          [0m| 10.7M/927M [00:09<62:29:31, 4.07kB/s]
  1%|[32m          [0m| 10.7M/927M [00:09<61:18:13, 4.15kB/s]
  1%|[32m          [0m| 10.7M/927M [00:09<63:45:17, 3.99kB/s]
  1%|[32m          [0m| 10.7M/927M [00:09<63:45:17, 3.99kB/s]
  1%|[32m          [0m| 10.7M/927M [00:10<66:42:30, 3.81kB/s]
  1%|[32m          [0m| 10.7M/927M [00:10<70:42:12, 3.60kB/s]
  1%|[32m          [0m| 10.7M/927M [00:10<70:42:12, 3.60kB/s]
  1%|[32m          [0m| 10.7M/927M [00:10<72:27:40, 3.51kB/s]
  1%|[32m          [0m| 10.7M/927M [00:10<75:28:30, 3.37kB/s]
  1%|[32m          [0m| 10.7M/927M [00:10<75:28:30, 3.37kB/s]
  1%|[32m          [0m| 10.7M/927M [00:11<80:37:25, 3.16kB/s]
  1%|[32m          [0m| 10.7M/927M [00:11<81:39:05, 3.12kB/s]
  1%|[32m          [0m| 10.7M/927M [00:11<81:39:05, 3.12kB/s]
  1%|[32m          [0m| 10.7M/927M [00:11<82:42:48, 3.08kB/s]
  1%|[32m          [0m| 10.7M/927M [00:11<83:58:49, 3.03kB/s]
  1%|[32m          [0m| 10.7M/927M [00:11<88:14:24, 2.88kB/s]
  1%|[32m          [0m| 10.7M/927M [00:12<88:14:24, 2.88kB/s]
  1%|[32m          [0m| 10.7M/927M [00:12<92:12:08, 2.76kB/s]
  1%|[32m          [0m| 10.7M/927M [00:12<95:30:48, 2.66kB/s]
  1%|[32m          [0m| 10.7M/927M [00:12<97:32:39, 2.61kB/s]
  1%|[32m          [0m| 10.8M/927M [00:13<99:31:00, 2.56kB/s]
  1%|[32m          [0m| 10.8M/927M [00:13<101:10:39, 2.51kB/s]
  1%|[32m          [0m| 10.8M/927M [00:13<101:10:39, 2.51kB/s]
  1%|[32m          [0m| 10.8M/927M [00:13<102:38:43, 2.48kB/s]
  1%|[32m          [0m| 10.8M/927M [00:13<101:26:40, 2.51kB/s]
  1%|[32m          [0m| 10.8M/927M [00:13<99:36:36, 2.55kB/s] 
  1%|[32m          [0m| 10.8M/927M [00:14<99:13:55, 2.56kB/s]
  1%|[32m          [0m| 10.8M/927M [00:14<92:07:28, 2.76kB/s]
  1%|[32m          [0m| 10.8M/927M [00:14<92:07:28, 2.76kB/s]
  1%|[32m          [0m| 10.8M/927M [00:14<90:15:24, 2.82kB/s]
  1%|[32m          [0m| 10.8M/927M [00:15<92:10:20, 2.76kB/s]
  1%|[32m          [0m| 10.8M/927M [00:15<95:56:00, 2.65kB/s]
  1%|[32m          [0m| 10.8M/927M [00:15<95:56:00, 2.65kB/s]
  1%|[32m          [0m| 10.8M/927M [00:15<100:28:53, 2.53kB/s]
  1%|[32m          [0m| 10.8M/927M [00:15<100:28:53, 2.53kB/s]
  1%|[32m          [0m| 10.8M/927M [00:15<101:18:08, 2.51kB/s]
  1%|[32m          [0m| 10.8M/927M [00:16<101:56:38, 2.50kB/s]
  1%|[32m          [0m| 10.8M/927M [00:16<102:17:31, 2.49kB/s]
  1%|[32m          [0m| 10.8M/927M [00:16<103:11:01, 2.47kB/s]
  1%|[32m          [0m| 10.8M/927M [00:16<101:16:21, 2.51kB/s]
  1%|[32m          [0m| 10.8M/927M [00:33<409:00:07, 622B/s]  
  1%|[32m          [0m| 10.8M/927M [00:33<624:32:15, 407B/s]
  1%|[32m          [0m| 10.8M/927M [00:34<624:32:15, 407B/s]
  1%|[32m          [0m| 10.8M/927M [00:34<779:22:35, 326B/s]
  1%|[32m          [0m| 10.8M/927M [00:34<888:11:34, 286B/s]
  1%|[32m          [0m| 10.9M/927M [00:34<965:53:13, 263B/s]
  1%|[32m          [0m| 10.9M/927M [00:35<1019:43:44, 250B/s]
  1%|[32m          [0m| 10.9M/927M [00:35<1057:12:18, 241B/s]
  1%|[32m          [0m| 10.9M/927M [00:35<1081:34:46, 235B/s]
  1%|[32m          [0m| 10.9M/927M [00:43<1081:34:46, 235B/s]
  1%|[32m          [0m| 10.9M/927M [00:43<938:16:57, 271B/s] 
  1%|[32m          [0m| 10.9M/927M [00:43<837:48:45, 304B/s]
  1%|[32m          [0m| 10.9M/927M [00:43<764:09:29, 333B/s]
  1%|[32m          [0m| 10.9M/927M [00:44<712:18:50, 357B/s]
  1%|[32m          [0m| 10.9M/927M [00:44<675:31:51, 377B/s]
  1%|[32m          [0m| 10.9M/927M [00:44<635:35:02, 400B/s]
  1%|[32m          [0m| 10.9M/927M [00:44<635:35:02, 400B/s]
  1%|[32m          [0m| 10.9M/927M [00:45<624:43:31, 407B/s]
  1%|[32m          [0m| 10.9M/927M [00:45<365:22:47, 696B/s]
  1%|[32m          [0m| 10.9M/927M [00:45<365:22:47, 696B/s]
  1%|[32m          [0m| 10.9M/927M [01:02<544:32:09, 467B/s]
  1%|[32m          [0m| 10.9M/927M [01:02<720:50:41, 353B/s]
  1%|[32m          [0m| 10.9M/927M [01:02<842:40:08, 302B/s]
  1%|[32m          [0m| 10.9M/927M [01:02<842:40:08, 302B/s]
  1%|[32m          [0m| 10.9M/927M [01:02<927:35:34, 274B/s]
  1%|[32m          [0m| 10.9M/927M [01:03<986:48:48, 258B/s]
  1%|[32m          [0m| 10.9M/927M [01:03<1028:35:51, 247B/s]
  1%|[32m          [0m| 10.9M/927M [01:03<1028:35:51, 247B/s]
  1%|[32m          [0m| 10.9M/927M [01:03<1060:08:48, 240B/s]
  1%|[32m          [0m| 10.9M/927M [01:04<1081:45:22, 235B/s]
  1%|[32m          [0m| 10.9M/927M [01:04<794:54:54, 320B/s] 
  1%|[32m          [0m| 10.9M/927M [01:04<593:59:02, 428B/s]
  1%|[32m          [0m| 11.0M/927M [01:04<453:44:22, 561B/s]
  1%|[32m          [0m| 11.0M/927M [01:07<355:25:37, 716B/s]
  1%|[32m          [0m| 11.0M/927M [01:07<332:19:10, 766B/s]
  1%|[32m          [0m| 11.0M/927M [01:07<316:15:43, 804B/s]
  1%|[32m          [0m| 11.0M/927M [01:08<289:36:16, 878B/s]
  1%|[32m          [0m| 11.0M/927M [01:08<282:00:59, 902B/s]
